src/DIRAC/FrameworkSystem/scripts/dirac_login.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
```

Comment:
Another one to add to the setup.cfg, I'll stop commenting on others

Reply:
yes i understood. I will add all new commands